common/server-sent-events.conf

```diff
@@ -5,4 +5,3 @@ proxy_cache off;
 # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
```

Comment:
Can we please link to https://serverfault.com/questions/801628/for-server-sent-events-sse-what-nginx-proxy-configuration-is-appropriate/801629#801629 instead? This is a much more detailed SO answer and it has some links to other sources too.